pkg/cmd/gen_driftignore_test.go

```diff
@@ -120,7 +103,7 @@ func TestGenDriftIgnoreCmd_ValidFlags(t *testing.T) {
 		{args: []string{"gen-driftignore", "--exclude-missing"}},
 		{args: []string{"gen-driftignore", "--exclude-changed"}},
 		{args: []string{"gen-driftignore", "--exclude-changed=false", "--exclude-missing=false", "--exclude-unmanaged=true"}},
-		{args: []string{"gen-driftignore", "--input", "/dev/stdin"}},
+		{args: []string{"gen-driftignore", "--input", "-"}},
```

Comment:
I think it will be more valuable if we replace this with `{args: []string{"gen-driftignore"}},` instead. We'd better want to cover the default value behavior instead if enforcing it explicitly.

Reply:
Good idea, if we cover that default and explicit `-`, that'll satisfy this note and the one below right?